hybridse/src/udf/udf_registry.h

```diff
@@ -1225,6 +1265,20 @@ class ExternalFuncRegistryHelper : public UdfRegistryHelper {
         return *this;
     }
 
+    template <typename Ret, typename... Args>
+    ExternalFuncRegistryHelper& with_return_args(
```

Comment:
this likely break existing auto type infer facility .  Consider support tuple type directly, or add clear note